### PR TITLE
Fix linting issues and a few bugs in core.scss

### PR DESCRIPTION
**Pull request by hharchani**

### CODE REVIEW

File: .css-lint.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ linters:
         enabled: false
     SelectorDepth:
         max_depth: 4
+    NestingDepth:
```
Comment (nisargjhaveri) [inline .css-lint.yml]: CSS me nesting kya hoti hai?

**Comment by polybuildr** [comment]
Apparently, he meant `master.scss`, not master. :P

/cc @nisargjhaveri

File: .css-lint.yml (hunk 1 above)
Comment (polybuildr) [inline .css-lint.yml]: Point. :P Sorry, @hharchani.

**Comment by nisargjhaveri** [comment]
Why do we have `master.scss` :'( :P

**Comment by polybuildr** [comment]
Because apparently, Atom autosuggested that name in a snippet. But I spoke to him, he'll probably change it to `core.scss`.

**Comment by polybuildr** [comment]
Skimmed through. This PR looks okay, though the rest of the directly committed content, I dunno. :P

**Pull request merged**